pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/block/ChunkPosition.kt

```diff
@@ -14,6 +14,9 @@ class ChunkPosition(world: World?, val x: Int, val z: Int) {
     val asLong: Long
         get() = (x.toLong() shl 32) or (z.toLong() and 0xFFFFFFFFL)
 
+    val chunk: Chunk?
+            = world?.getChunkAt(x, z)
```

Comment:
```suggestion
    val chunk: Chunk?
            get() = world?.getChunkAt(x, z)
```
I think you meant to do this?